iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -17,7 +17,9 @@
 #ifndef IOX_HOOFS_CXX_HELPLETS_HPP
 #define IOX_HOOFS_CXX_HELPLETS_HPP
 
-#include <assert.h>
+#include "iceoryx_hoofs/cxx/type_traits.hpp"
```

Comment:
This is part of another PR, so please ignore helplets and type_traits changes. They will disappear once the other PR is merged